bigtable-dataflow-parent/bigtable-beam-import/src/test/resources/com/google/cloud/bigtable/beam/hbasesnapshots/EndToEndIT-snapshot-generate.sh

```diff
@@ -102,32 +109,18 @@ put 'test','99', 'cf:a', 'value99', 100
 put 'test','100', 'cf:a', 'value100', 100
 snapshot 'test', 'test-snapshot'
 list_snapshots
+EOF
```

Comment:
@TracyCuiCan You need to add the compressed table here. We will also need to replace the zip file with the one with both snapshots.